drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/enrollment_attribute_calculations.rb

```diff
@@ -214,7 +214,8 @@ def return_dates_for_batch(filter, batch)
       def enrollment_data
         # Source ProjectIDs are used in the report
         project_ids_from_groups = GrdaWarehouse::Hud::Project.where(id: filter.effective_project_ids_from_secondary_project_groups).pluck(:project_id)
-        universe.members.where(a_t[:project_id].in(project_ids_from_groups)).map(&:universe_membership)
+        member_ids = universe.members.where(a_t[:project_id].in(project_ids_from_groups)).pluck(:universe_membership_id)
+        Enrollment.find(member_ids)
```

Comment:
I'm not a fan of this syntax for multi-find, since generally `find` would return a single enrollment.  Can we do `Enrollment.where(id: member_ids).to_a` or something more clear?  Also, are `universe_membership_id` really `Enrollment.id`?

Reply:
Sure... And yes, `SimpleReports.universe_menbership_id` is a polymorphic pointer to the report-specific class, which in this case is `AllNeighborsSystemDashboard::Enrollment`.